src/arch/devlog.cpp

```diff
@@ -98,7 +100,7 @@ void logMessage(E_MsgLevel paLevel, const char *paMessage, ...) {
 
 void printLogMessage(E_MsgLevel paLevel, const char *paMessage) {
 #ifndef __ZEPHYR__
-  (paLevel == E_MsgLevel::Error ? std::cerr : std::cout) << scLogLevel[static_cast<int>(paLevel)] << ": " << getNanoSecondsMonotonic() << ": " << paMessage;
+  (paLevel == E_MsgLevel::Error ? std::cerr : std::cout) << scLogLevel[static_cast<int>(paLevel)] << ": " << getRealtimeString().c_str() << ": " << paMessage;
```

Comment:
Can you explain why you are explicitly calling .c_str() for the output? Concatinating the std::string object should work out of the box.

Reply:
Thank you. Just fixed it.